apps/app/src/views/routes/Settings/components/users/UserItemActions.tsx

```diff
@@ -63,6 +83,54 @@ export const UserItemActions: React.FC<UserItemActionsProps> = ({ user }) => {
           <MenuItem icon={<FiEdit fontSize="1.2em" />} onClick={onOpen}>
             Edit Roles
           </MenuItem>
+          <MenuItem
+            icon={<FiTrash fontSize="1.3em" color="red" />}
+            disabled={deleteLoading}
+            onClick={async () => {
```

Comment:
Can we extract this to a method higher up and use `useCallback`. We should just default get in the habit of this. 

See https://semaphoreci.com/blog/react-usecallback-hook#:~:text=Benefits%20of%20using%20the%20useCallback%20hook&text=Restricting%20rendering%20of%20child%20components,as%20props%20to%20child%20components.